packages/wallet/test/services/util/equals.test.ts

```diff
@@ -26,10 +26,10 @@ describe('equals', () => {
     expect(strictEquals('1', '1')).toBe(true);
   });
 
-  test('arrayEquals', () => {
-    expect(arrayEquals([], [], strictEquals)).toBe(true);
-    expect(arrayEquals(['a'], ['a', 'b'], strictEquals)).toBe(false);
-    expect(arrayEquals(['a', 'b'], ['a', 'b'], strictEquals)).toBe(true);
+  test('sameArrayItems', () => {
+    expect(sameArrayItems([], [], strictEquals)).toBe(true);
+    expect(sameArrayItems(['a'], ['a', 'b'], strictEquals)).toBe(false);
+    expect(sameArrayItems(['a', 'b'], ['a', 'b'], strictEquals)).toBe(true);
```

Comment:
This request is out of scope. It would be nice to have a test for:
```diff
+ expect(sameArrayItems(['a', 'b'], ['b', 'a'], strictEquals)).toBe(true);
```

Reply:
ce2df04debff48b03aa9a0d75309cfa2ca8b0163